apps/drained/app.ts

```diff
@@ -1,9 +1,9 @@
 const app = "drained";
 
 // from boot.js
-declare var drainedInterval: IntervalId | undefined;
-if(typeof drainedInterval !== "undefined")
-  drainedInterval = clearInterval(drainedInterval) as undefined;
+const globals: { drainedInterval: IntervalId | undefined } = global as any;
+if(typeof globals.drainedInterval !== "undefined")
+  globals.drainedInterval = clearInterval(globals.drainedInterval) as undefined;
```

Comment:
The `declare var` indicates that this is a global access, so I would prefer that we don't modify this